dxp/taglib_aui_input.md

```diff
@@ -0,0 +1,1409 @@
+	</div>
+</div>
+```
+
+_html_
+
+```html
+<div class="clay-range clay-range-progress-none">
+	<label class="form-control-label">
+		<div class="form-control-label-text">
+			Range
+		</div>
+		<div class="clay-range-input">
+			<input
+				class="field form-control-range form-control"
+				id="_portlet_namespace_clayRange"
+				max="100"
+				min="0"
+				name="_portlet_namespace_clayRange"
+				title="clay-range"
+				type="range"
+				value="75"
+				step="1"
+			/>
+			<div class="clay-range-track"></div>
+			<div class="clay-range-progress">
+				<div class="clay-range-thumb"></div>
+			</div>
+		</div>
+	</label>
+</div>
+```
+
+## Resource
+
+This `aui:input` type outputs a `readonly` text `input` with JavaScript that highlights all text inside on click. This type passes the attributes `id`, `title`, and `value` to the `liferay-ui:input-resource` tag which outputs the `readonly` input. The tag `liferay-ui:input-resource` always namespaces the `id`. The attribute `useNamespace="<%= false %>"` does nothing here.
+
+| Attributes | Description                                                                |
+| ---------- | -------------------------------------------------------------------------- |
+| `cssClass` | Adds classes to `<input class="form-control" />`                           |
+| `id`       | A `string` to use as the `id`. This is always namespaced.                  |
+| `name`     | Same as `id`. This attribute is not output by `liferay-ui:input-resource`. |
+| `title`    | A `string` to use in the `title` attribute.                                |
+| `value`    | A `url` that gets passed to the `liferay-ui:input-resource` tag.           |
+
+_jsp_
+
+```jsp
+<aui:input label="Cadetblue" name="cadetblue" type="resource" />
+```
+
+_html_
+
+```html
+<div class="form-group input-resource-wrapper">
+	<label class="control-label" for="_portlet_namespace_cadetblue">
+		Cadetblue
+	</label>
+	<input
+		class="form-control form-text lfr-input-resource "
+		id="_portlet_namespace_cadetblue"
+		onclick="this.select();"
+		readonly="true"
+		type="text"
+		value="null"
+	/>
+</div>
+```
+
+## Time Zone
+
+The `aui:input` type `timeZone` outputs a `select` element containing time zones from around the world. The attributes in the `aui:input` get passed to the tag `liferay-ui:input-time-zone` which outputs a `<select>` element with `<options>`.
+
+| Attributes     | Description                                                                                                                    |
+| -------------- | ------------------------------------------------------------------------------------------------------------------------------ |
+| `cssClass`     | Adds classes to `<select class="form-control" />`.                                                                             |
+| `daylight`     | TODO: `daylight="true"` What does this do?                                                                                     |
```

Comment:
Nothing. It is not used in the JSP, nor any other place of DXP's code. Simply mark it as deprecated or unused.